oio/rdir/client.py

```diff
@@ -85,6 +84,58 @@ def get_assignation(self, **kwargs):
                                  rawx['addr'], exc)
         return all_rawx, all_rdir
 
+    def assign_all_meta2(self, max_per_rdir=None, **kwargs):
```

Comment:
We could. However if there needs to be a change down the line, it'll be easier and probably safer.
Should I factorize it anyway ?

Reply:
If there needs to be a change, only parts of this method will be changed, not the whole method. It's a bad practice to duplicate code "just in case". Maybe the method is already too long and should be split. Unless you implement a different logic in this pull-request, yes please, factorize it.